.github/workflows/build.yml

```
@@ -115,6 +116,69 @@ jobs:
           leeway build dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}"
           echo "previewctl_hash=$(leeway describe dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}" -t '{{ .Metadata.Version }}')" >> $GITHUB_OUTPUT
 
+  update_pr_desc:
+    name: Update PR summary
+    needs: [ configuration ]
```

You only want this to run if we create a preview env and we deploy to it successfully. Also move it below the `install:` job.

```suggestion
    needs: [ configuration,  infrastructure, install ]
```